srcDoc/release_notes.rst

```diff
@@ -168,6 +168,12 @@ Ensure compatibility for the Classic C++ Implementation (#114)
 Some of the changes added for #55 broke compatibility when compiling certain
 platforms with no support for C++11. This issue has been fixed.
 
+Run RTI Perftest easier in VxWorks (#167)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Make this line match the chars in the tittle (else it gives warnings when generating the html (the generator is quite picky).